Lib/textwrap.py

```diff
@@ -73,41 +89,39 @@ class TextWrapper:
     # (after stripping out empty strings).
     word_punct = r'[\w!"\'&.,?]'
     letter = r'[^\d\W]'
-    whitespace = r'[%s]' % re.escape(_whitespace)
-    nowhitespace = '[^' + whitespace[1:]
-    wordsep_re = re.compile(r'''
+    whitespace = fr'[{_whitespace}]'
```

Comment:
re.escape is a no-op here?

Reply:
It prefixes each character with a backslash, but none of the whitespace characters is a metacharacter for `sre` / the regular expression engine, so in practice yes it is a no-op.

A